src/repositories/userRepository.ts

```diff
@@ -1,6 +1,8 @@
 import { User, UserRole } from '../entities/user';
 
-export const findAdminUserByEmail = async (email): Promise<User | null> => {
+export const findAdminUserByEmail = async (
+  email,
```

Comment:
specify type of `email` in inout parameters

Reply:
done!